ci_cd/tasks/setver.py

```diff
@@ -14,15 +14,15 @@
 
```

Comment:
from __future__ import annotations , is not used anywhere

Reply:
True. It is a generic import to ensure annotations can be written without quotations and more. [Here](https://stackoverflow.com/a/61544901/12404091)'s a nice explanation from StackOverflow.